### DISCUSSION TITLE
Feature Request: Copy Code

### DISCUSSION THREAD

**Discussion by jvanderaa** [mmistakes/minimal-mistakes#2766] [Ideas]
I'm looking to add a "copy code" button on top of the liquid tags:

```
{% highlight yaml linenos %}
{% raw %}

```

I've attempted a few guides:

https://experimentingwithcode.com/creating-a-jekyll-blog-with-bootstrap-4-and-sass-part-4/
https://www.aleksandrhovhannisyan.com/blog/dev/how-to-add-a-copy-to-clipboard-button-to-your-jekyll-blog/

Both "out of the box" are not functional when running locally. I was able to get a portion working with copy code if I remove the linenos. Otherwise with linenos setup (which I absolutely love in the more recent versions) it copies the entire table including the line numbers. Not sure if there is an elegant solution there.

I may just need to dig in a bit more on the JS front of things to get it truly working, but thought I'd see if there was interest of just adding it to the base of Minimal Mistakes all together.

**Comment by dieghernan** [comment]
See https://github.com/mmistakes/minimal-mistakes/discussions/2795
